web/test/mock/config.js

```diff
@@ -18,6 +18,11 @@
         },
         record: {
             limit: 50
+        },
+        recordType: {
+            visible: false,
```

Comment:
Should be `display` here too (or change `web-config-js` template to use `visible`